### ISSUE TITLE
Not updating on resize or orientation change

### ISSUE THREAD

**Issue by karlskidmore** [codemonk-digital/vue-responsiveness#21]
This lib looks great, just what I've been looking for.  Not sure if this is a bug or by design, but...

**Describe the bug**
Exposed computed values are not updating on resize or orientation change, only a full refresh of the page updates the values.  Tested on desktop and iphone.

**To Reproduce**
1. Add below code to page:
```
    $matches.sm.min -> {{ $matches.sm.min }}  <br>
    $matches.sm.max -> {{ $matches.sm.max }}  <br>
    $matches.sm.only -> {{ $matches.sm.only }}  <br>
```
2. Resize browser / change orientation of phone

**Expected behavior**
$matches values to update as browser resized / phone's orientation changed

**Desktop (please complete the following information):**
 - MacOS
 - Chrome
 - 121

**Smartphone (please complete the following information):**
 - iPhone8
 - iOS16.7.4
 - safari

**Comment by andrei-gheorghiu** [comment]
@karlskidmore, my apologies for not responding earlier. 
I wasn't getting updates from this repo since I moved it to `codemonk-digital` space. 

Would you be able to share your `package.json`? (I'm interested in the `vue` and `vue-responsiveness` versions used. If it makes sense, use codesandbox.io or similar to create a reproducible example (and share the relevant parts of your config).

**Comment by andrei-gheorghiu** [comment]
Apparently, there was a breaking change in Vue's reactivity tracking which affects how this plugin is tracking changes, creating an infinite loop. I'm getting an `Uncaught RangeError: Maximum call stack size exceeded` when using `vue@^3.2` with `vue-responsiveness@^1.3`. Downgrading the plugin might fix it, but I haven't yet investigated which are the highest compatible versions, I'm aiming to make it work in latest `vue` anyways.

**Comment by karlskidmore** [comment]
> @karlskidmore, my apologies for not responding earlier. I wasn't getting updates from this repo since I moved it to `codemonk-digital` space.

Hi Andrei, hey thanks for getting back and thanks again for producing this repo.
 
> Would you be able to share your `package.json`? (I'm interested in the `vue` and `vue-responsiveness` versions used). If it makes sense, use codesandbox.io or similar to create a reproducible example (and share the relevant parts of your config).
I wish it were that easy.  Mine is a Spring Boot MPA project pulling dependencies from webjars, so no SPA, no bundling, etc, and no package.json.  I'm still using Vue 3.2.47 if that helps.

> As a side-note, this was tested on both mac and iPhone and was working fine a few versions ago but currently I no longer have a mac. I'll try to use BrowserStack, hoping I'll be able to repro.
I'm happy to re-test locally if/when needed.

Thanks again.

**Comment by andrei-gheorghiu** [comment]
I was able to figure the problem out and found a workaround. Unfortunately, it will take 2-3 days more as I discovered my npm account was 2FA-bound to the mac I mentioned earlier and I'm currently in the process of resetting 2FA (which, according to the  `npm` response, will take 2-3 working days). As soon as I recover it, I'll release `1.7`.
Meanwhile you can test if it's working by specifying this in package.json:
```"vue-responsiveness": "https://github.com/codemonk-digital/vue-responsiveness.git#main"```
, as I've already pushed the fix on `main`.

**Comment by andrei-gheorghiu** [comment]
@karlskidmore, the issue should be fixed in `v0.2.0`, which was published a few seconds ago. I'll close this as fixed. If the issue is not fixed, feel free to reopen it.
